packages/executor/src/interfaces.ts

```diff
@@ -115,6 +116,10 @@ export interface NetworkConfig {
   // etherscan api is used to fetch gas prices
   // default = "" (empty string)
   etherscanApiKey: string;
+  // enables contidional rpc
+  conditionalTransactions: boolean;
```

Comment:
Why is conditional expressed as a boolean? Can't we use this endpoint by default on all chains that support it?